### PR TITLE
updated base path for ansible

### PR DESCRIPTION
**Pull request by dimleevik**
##### SUMMARY

Updated base path for ansible collection

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

plugins

### CODE REVIEW

**Review by sdodsley — DISMISSED** [review]

File: plugins/module_utils/fusion.py [hunk 1]
```diff
@@ -102,7 +104,8 @@ def get_fusion(module):
     key_file = module.params[PARAM_PRIVATE_KEY_FILE]
 
     config = fusion.Configuration()
-    config.host = environ.get(ENV_API_HOST, environ.get(ENV_HOST, config.host))
+    if ENV_API_HOST in environ:
```
Comment (vromash) [inline plugins/module_utils/fusion.py:107]: why `ENV_HOST` was removed? it should be in use until we deprecate it in 2.0.0

**Review by andreepyro — CHANGES_REQUESTED** [review]
Please leave ENV_HOST variable in place.

**Review by jankodera-ps — APPROVED** [review]

**Review by vromash — DISMISSED** [review]

**Comment by vromash** [comment]
don't we need changelog for that?

**Review by sdodsley — APPROVED** [review]

File: changelogs/127_change_logic_of_fusion_api_host.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - FUSION_API_HOST && FUSION_HOST - changed logic, now this variables require host name without path
```
[the remark this answers is not in the document]
Comment (DenDen039) [reply changelogs/127_change_logic_of_fusion_api_host.yml:2]: Or we just need to document somewhere that you can specify Fusion host parameter. As well as tex field

**Review by jankodera-ps — APPROVED** [review]

**Review by DenDen039 — APPROVED** [review]

**Comment by jankodera-ps** [comment]
@andreepyro Were the requested changes met?

**Review by andreepyro — APPROVED** [review]

**Comment by andreepyro** [comment]
> @andreepyro Were the requested changes met?

@dimleevik @jankodera-ps Sorry, didn't notice the new commits. Yes, look good to me now :)

**Pull request merged**